ui/admin/app/styles/app.scss

```diff
@@ -452,3 +454,103 @@
     }
   }
 }
+
+// secret editor
+.json-secret-editor {
+  .CodeMirror {
+    height: 210px;
+  }
+
+  .CodeMirror-gutters {
+    background: #31332b;
+  }
+}
+
+.secret-editor {
+  background: #272822;
+  border-radius: 4px;
+  height: 210px;
+  margin-bottom: sizing.rems(l);
+}
+
+.skeleton-wrapper {
+  color: var(--ui-gray);
+  display: flex;
+  flex-direction: row;
+  position: relative;
+}
+
+.skeleton-gutter {
+  background: #31332b;
+  border-radius: 4px 0 0 4px;
+  height: 210px;
+  width: 52px;
+}
+
+.skeleton-body {
+  width: 100%;
+
+  > div {
+    background: #31332b;
+    border-radius: 16px;
+    height: 8px;
+    margin: 10.8px;
+
+    &:first-child {
+      margin-top: 16px;
+    }
+
+    &.s {
+      width: 5%;
+    }
+
+    &.m {
+      width: 10%;
+    }
+
+    &.l {
+      width: 15%;
+    }
+
+    &.xl {
+      width: 20%;
+    }
+  }
+}
+
+.details-view {
+  font-size: 14px;
+  left: 50%;
+  margin: -12px 0 0 -165.5px;
+  position: absolute;
+  top: 50%;
+  z-index: 1;
+
+  .rose-icon {
+    --size: sizing.rems(m);
+  }
+}
+
+.edit-view {
+  left: 50%;
+  margin: -11px 0 0 -100px;
+  position: absolute;
+  top: 50%;
+  z-index: 1;
+
+  .rose-button-body {
+    font-size: 14px;
+  }
+}
+
+.rose-button-secret-editor-button {
+  background: #272822;
+  border: 1px solid #6f7682;
+  border-radius: 2px;
+  box-shadow: 0 4px 4px #000004;
+  height: 36px;
+  left: 50%;
```

Comment:
Are these `left` and `top` necessary? Aren't they already being handled by this button's wrapper?

Reply:
You are correct. Removed.